target/linux/ipq40xx/files-6.6/arch/arm/boot/dts/qcom/qcom-ipq4019-wia3300-20.dts

```diff
@@ -0,0 +1,447 @@
+// SPDX-License-Identifier: GPL-2.0-or-later OR MIT
+
+#include "qcom-ipq4019.dtsi"
+#include <dt-bindings/gpio/gpio.h>
+#include <dt-bindings/input/input.h>
+#include <dt-bindings/leds/common.h>
+#include <dt-bindings/soc/qcom,tcsr.h>
+
+/ {
+	compatible = "skspruce,wia3300-20", "qcom,ipq4019";
+	model = "SKSpruce WIA3300-20";
+
+	aliases {
+		label-mac-device = &gmac;
+		led-boot = &led_status_red;
+		led-failsafe = &led_status_lime;
+		led-running = &led_status_lime;
+		led-upgrade = &led_status_red;
+	};
+
+	chosen {
+		stdout-path = "serial0:115200n8";
+	};
+
+	keys {
+		compatible = "gpio-keys";
+
+		pinctrl-0 = <&key_pins>;
+		pinctrl-names = "default";
+
+		reset {
+			label = "reset";
+			gpios = <&tlmm 18 (GPIO_ACTIVE_LOW|GPIO_PULL_DISABLE)>;
```

Comment:
Does `GPIO_PULL_DISABLE` do anything?

Reply:
I did some tests, but unfortunately it didn't take effect. I will remove them.